src/esp/sensor/CameraSensor.cpp

```diff
@@ -66,18 +67,9 @@ void CameraSensor::recomputeBaseProjectionMatrix() {
   recomputeProjectionMatrix();
 }  // CameraSensor::recomputeNearPlaneSize
 
-CameraSensor& CameraSensor::setProjectionMatrix(
-    gfx::RenderCamera& targetCamera) {
-  targetCamera.setProjectionMatrix(width_, height_, projectionMatrix_);
-  return *this;
-}
-
-CameraSensor& CameraSensor::setTransformationMatrix(
-    gfx::RenderCamera& targetCamera) {
-  CORRADE_ASSERT(!scene::SceneGraph::isRootNode(targetCamera.node()),
-                 "CameraSensor::setTransformationMatrix: target camera cannot "
-                 "be on the root node of the scene graph",
-                 *this);
+Magnum::Matrix4 CameraSensor::computeTransformationMatrix() {
```

Comment:
I think this entire function no longer needs to exist.  This was a thing when we needed to apply the transform of a given sensor to the camera, but now `renderCamera_->node() == this->node()` so this function should be a no-op.